<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>461</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputTextEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outputText"/>
     </item>
     <item>
      <widget class="QPushButton" name="modeButton">
       <property name="text">
        <string>ModeButton</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clearAllButton">
         <property name="text">
          <string>ClearAll</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertButton">
         <property name="text">
          <string>Convert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="instructionLabel">
       <property name="text">
        <string>Instruction Label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="warningLabel">
       <property name="text">
        <string>text label</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMorse_Translator">
    <property name="title">
     <string>Morse Translator</string>
    </property>
   </widget>
   <addaction name="menuMorse_Translator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>outputText</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
